cilium monitor: don't throw panic when it cannot find monitor

Bail out with just a normal error msg instead of panic + trace:
```
# cilium monitor -D -d
panic: Unable to get object: no such file or directory

goroutine 1 [running]:
panic(0xdaa880, 0xc421385670)
	/usr/lib/golang/src/runtime/panic.go:500 +0x1a1
github.com/cilium/cilium/cilium/cmd.runMonitor()
	/usr/lib/golang/src/github.com/cilium/cilium/cilium/cmd/monitor.go:183 +0x427
github.com/cilium/cilium/cilium/cmd.glob..func14(0x163e080, 0xc421413da0, 0x0, 0x2)
	/usr/lib/golang/src/github.com/cilium/cilium/cilium/cmd/monitor.go:42 +0x14
github.com/cilium/cilium/vendor/github.com/spf13/cobra.(*Command).execute(0x163e080, 0xc421413b40, 0x2, 0x2, 0x163e080, 0xc421413b40)
	/usr/lib/golang/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:648 +0x443
github.com/cilium/cilium/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0x163b820, 0xc420066058, 0x0, 0x1)
	/usr/lib/golang/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:735 +0x367
github.com/cilium/cilium/vendor/github.com/spf13/cobra.(*Command).Execute(0x163b820, 0x0, 0x0)
	/usr/lib/golang/src/github.com/cilium/cilium/vendor/github.com/spf13/cobra/command.go:693 +0x2b
github.com/cilium/cilium/cilium/cmd.Execute()
	/usr/lib/golang/src/github.com/cilium/cilium/cilium/cmd/root.go:69 +0x31
main.main()
	/usr/lib/golang/src/github.com/cilium/cilium/cilium/main.go:20 +0x14

```

--- a/cilium/cmd/monitor.go
+++ b/cilium/cmd/monitor.go
@@ -178,7 +178,8 @@ func runMonitor() {
 
 	events, err := bpf.NewPerCpuEvents(&eventConfig)
 	if err != nil {
-		panic(err)
+		fmt.Printf("Error: Unable to get BPF events (%s)\n", err)
+		os.Exit(1)
 	}
 
 	if eventType != "" {

--- a/pkg/bpf/bpf.go
+++ b/pkg/bpf/bpf.go
@@ -267,7 +267,7 @@ func ObjGet(pathname string) (int, error) {
 	)
 
 	if fd == 0 || err != 0 {
-		return 0, fmt.Errorf("Unable to get object: %s", err)
+		return 0, fmt.Errorf("Unable to get object %s: %s", pathname, err)
 	}
 
 	return int(fd), nil

